apps/daimo-mobile/src/view/screen/send/SendScreen.tsx

```diff
@@ -64,36 +70,40 @@ export default function SendScreen({ route }: Props) {
         bounces={false}
         keyboardShouldPersistTaps="handled"
       >
-        {recipient && (
-          <SetAmount recipient={recipient} dollars={dollars || 0} />
-        )}
-        {!recipient && (
-          <>
-            {tab !== "createNote" && (
-              <SegmentedControlFixed
-                values={tabs}
-                selectedIndex={tab === "scan" ? 1 : 0}
-                onValueChange={setSegmentVal}
-                fontStyle={{ fontSize: 16 }}
-                activeFontStyle={{ fontSize: 16 }}
-                style={{ height: 40 }}
-              />
-            )}
-            {tab !== "createNote" && <Spacer h={16} />}
-            {tab === "search" && <SearchTab />}
-            {tab === "scan" && <ScanTab hide={search} />}
-            {tab === "createNote" && (
-              <SendAmountChooser
-                actionDesc={
-                  noteCreated ? "Payment link created" : "Creating payment link"
-                }
-                dollars={noteDollars}
-                onSetDollars={noteCreated ? undefined : setNoteDollars}
-                onCancel={noteCreated ? undefined : search}
-              />
-            )}
-          </>
-        )}
+        <KeyboardAvoidingView behavior="position" keyboardVerticalOffset={128}>
```

Comment:
lgtm. can you post a screenshot?

Reply:
Before and after
<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/72c74ecb-24e4-4110-9844-d720bbcc5c94">

<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/1badcda7-7007-4d44-9563-92b129ee50f5">